resources/website_common.xsl

```diff
@@ -134,8 +134,19 @@
 
   <xsl:template match="programlisting">
     <div class="pre_wrapper">
-        <xsl:apply-imports />
+      <xsl:apply-imports />
     </div>
+    <!-- Asciidoctor's COSOLE widget -->
```

Comment:
`s/COSOLE/CONSOLE/` seems worth fixing for greppability

Reply:
:+1: